asdf/tests/test_schema.py

```diff
@@ -617,51 +619,59 @@ def test_schema_resolved_via_entry_points():
     assert tag in repr(s)
 
 
-@pytest.mark.parametrize('use_numpy', [False, True])
-def test_large_literals(use_numpy):
-
-    largeval = 1 << 53
-    if use_numpy:
-        largeval = np.uint64(largeval)
+@pytest.mark.parametrize("num", [constants.MAX_NUMBER+1, constants.MIN_NUMBER-1])
+def test_max_min_literals(num):
 
     tree = {
-        'large_int': largeval,
+        'test_int': num,
     }
 
     with pytest.raises(ValidationError):
         asdf.AsdfFile(tree)
 
     tree = {
-        'large_list': [largeval],
+        'test_list': [num],
     }
 
     with pytest.raises(ValidationError):
         asdf.AsdfFile(tree)
 
     tree = {
-        largeval: 'large_key',
+        num: 'test_key',
     }
 
     with pytest.raises(ValidationError):
         asdf.AsdfFile(tree)
 
-    tree = {
-        'large_array': np.array([largeval], np.uint64)
-    }
 
-    ff = asdf.AsdfFile(tree)
-    buff = io.BytesIO()
-    ff.write_to(buff, auto_inline=None)
-
-    ff.set_array_storage(ff.tree['large_array'], 'inline')
-    buff = io.BytesIO()
+@pytest.mark.parametrize("num, ttype", [
+    (constants.MAX_NUMBER+1, "val"),
+    (constants.MAX_NUMBER+1, "list"),
+    (constants.MAX_NUMBER+1, "key"),
+    (constants.MIN_NUMBER-1, "val"),
+    (constants.MIN_NUMBER-1, "list"),
+    (constants.MIN_NUMBER-1, "key")
+])
```

Comment:
Since each `ttype` string is tested with each value of `num`, one can doubly-parametrize like so:

```python
@pytest.mark.parametrize("num", [constants.MAX_NUMBER+1, constants.MIN_NUMBER-1])
@pytest.mark.parametrize("ttype", ["val", "list", "key"])
def test_max_min_literals_write(num, ttype, tmpdir):
    ...
```

Reply:
I rewrote the parametrization to do this double parametrization.